src/arduino.cc/arduino-builder/main.go

```diff
@@ -134,7 +134,15 @@ var loggerFlag *string
 var versionFlag *bool
 var vidPidFlag *string
 
+var Usage = func(){
```

Comment:
declaration taken directly from [here](https://golang.org/pkg/flag/#pkg-variables), and edited as needed.